locale/fi/LC_MESSAGES/django.po

```diff
@@ -2108,6 +2113,18 @@ msgstr "Palautusta arvostellaan..."
 msgid "Current score"
 msgstr "Nykyiset pisteet"
 
+#: exercise/templates/exercise/_exercisecollection.html:25
+#, fuzzy
+#| msgid "Total points"
+msgid "Total score"
+msgstr "Yhteispisteet"
+
+#: exercise/templates/exercise/_exercisecollection.html:29
+#, fuzzy
+#| msgid "Teacher of the course"
+msgid "Score on this course"
+msgstr "Kurssin opettaja"
```

Comment:
I don't think  the automatic translation went so well :)

The `#, fuzzy` means that this was automatically translated from a string that was pretty close. Using a translation program helps to track which translations are final. Any case, check translations which include `fuzzy`, fix them and remove the two lines (fuzzy and source msgid).